README.md

```diff
@@ -76,7 +76,7 @@ or
 select a build you are interested in, and download the package linked in
 the "**Artifacts**" section.
 
```

Comment:
Suggest adding a note about needing a GitHub account (and being logged in) to download artifacts.

Reply:
Totally forgot about this section in README - thanks for pointing this out :)

Fixed in d2797f7a2eca3de36c2c420050bfa8691c3c2864.